db/migrate/20150819105725_add_lookup_value_match_to_host_and_hostgroup.rb

```diff
@@ -5,10 +5,20 @@ def up
     Host::Managed.reset_column_information
     Hostgroup.reset_column_information
 
-    Host::Managed.where(:type => 'Host::Managed').find_in_batches(:batch_size => 100) do |group|
-      group.each do |host|
-        host.update_attribute(:lookup_value_matcher, host.send(:lookup_value_match))
-      end
+    matchers = LookupValue.where("lookup_values.match like ?", "fqdn=%").group("lookup_values.match").count.keys
```

Comment:
I'll try. I think it didn't work for me when I did. as for the count - it goes with the group, resulting in a hash of matches and how many times they're seen (sort of like uniqueness on a field instead of an entire row)